doc/herbstluftwm.txt

```diff
@@ -835,6 +835,11 @@ new_attr *bool*|*color*|*int*|*string*|*uint* 'PATH' ['VALUE']::
     with +my_+. If 'VALUE' is supplied, then it is written to the attribute (if
     this fails the attribute still remains).
 
+[[watch]]watch 'PATH'::
```

Comment:
Why is watch twice in here? Is it syntax I don't know about?

Reply:
Also should we add the new command to NEWS?

Reply:
I've updated the NEWS file.
The `[[watch]]` defines a label and it's referenced in the `HOOKS` section later (yielding a clickable link in the html doc).

Reply:
Ah! I see, thanks!